### ISSUE TITLE
Tweak Suggestion - Drones

### ISSUE THREAD

**Issue by ZgSvDaCpke479pF6DzfLgMvK9AiyNsgUBKPghdh** [William758/ZetTweaks#7]
Allow all types of drones to be repaired after being destroyed, and let the TC-280 Prototype spawn in any stage rather than just Rallypoint Delta?

Merely for utility and variety purposes; drones become largely obsolete after a few stages, and the Prototype's presence is very minimal in the game in general.

**Comment by Dave247** [comment]
I had a instance where two TC280 spawned in a single level, is that intended?

**Comment by William758** [comment]
Yeah, all I've done is tell the game that it can spawn on the stage. The game then decides if and how many it wants to spawn.
